File: algorithms/06_larger_and_smaller_number.py
array = []

for i in range(3):
    array.append(int(input("Digite o %dº número: " % (i + 1))))

    if i != 0:
        if larger_number < array[i]:
            larger_number = array[i]

        else:
            smallest_number = array[i]
    else:
        larger_number = array[i]
        smallest_number = array[i]

print("O MAIOR número é: %d" % larger_number)
print("O MENOR número é: %d" % smallest_number)

File: algorithms/06_larger_and_smaller_number.c
#include <stdio.h>
int main()
{
    int array[3];
    int larger_number;
    int smallest_number;

    for (int i = 0; i < 3; i++)
    {
        printf("Digite o %dº número: ", (i + 1));
        scanf("%d", &array[i]);

        if (i != 0)
        {
            if (larger_number < array[i])
            {
                larger_number = array[i];
            }
            else
            {
                smallest_number = array[i];
            }
        }
        else
        {
            larger_number = array[i];
            smallest_number = array[i];
        }
    }

    printf("O MAIOR número é: %d\n", larger_number);
    printf("O MENOR número é: %d\n", smallest_number);

    return 0;
}